lib/node_modules/@stdlib/stats/incr/wstdev/examples/index.js

```diff
@@ -0,0 +1,41 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+var uniform = require( '@stdlib/random/base/uniform' );
+var incrwstdev = require( './../lib' );
+
+var accumulator;
+var s;
+var x;
+var w;
+var i;
+
+// Initialize an accumulator:
+accumulator = incrwstdev();
+
+// For each simulated datum, update the weighted sample standard deviation...
+console.log( '\nValue\tWeight\tSigma\n' );
```

Comment:
```suggestion
// Initialize an accumulator:
var accumulator = incrwstdev();

// For each simulated datum, update the weighted sample standard deviation...
console.log( '\nValue\tWeight\tSigma\n' );
var s;
var x;
var w;
var i;
```

Ideally, variables are declared in examples as closely as possible to where they are first assigned, but not within blocks.